src/modules/esl-forms/esl-select-list/README.md

```diff
@@ -1,13 +1,37 @@
 # [ESL](../../../../README.md) Select List
 
-Version: *1.0.0-beta*.  
-***Important Notice: the component is under beta version, it was tested and now ready to use but be aware of its potential critical API changes.***
+Version: *1.0.0*.  
 
 Authors: *Alexey Stsefanovich (ala'n)*.
 
 ESLSelectList - is a custom element, for customized selectable list representation.
 
-ESLSelectList is HTML5 form compatible.
-HTMLSelectElement is used as model for ESLSelectList in the current implementation.
+ESLSelectList is HTML5 form compatible as it decorates the native HTMLSelectElement under the hood.
+HTMLSelectElement is used as model for ESLSelect.
 
-Description TBD.
+### Attributes / Properties
+
+- `select-all-label` - select all options text
+
+- `disabled` - disabled state marker
+
+- `pin-selected` - marker to top pin selected items to top in dropdown
```

Comment:
Maybe replace by
`marker for selecting items to be pinned to the top of the dropdown`
?